### ISSUE TITLE
Trinity requires access to Java version 1.6 or 1.7

### ISSUE THREAD

**Issue by RxLoutre** [clemgoub/dnaPipeTE#5]
Hi ! 

I got a error while trying to test dnaPipeTE with the following command :` python3 ./dnaPipeTE.py -input test_dataset.fastq -output /home/loutre/dnaPipeTE/try -genome_size 10000000 -genome_coverage 0.1 -sample_number 1
`
Here the error message I get : 

```
Start time: Fri Jan 27 10:22:45 2017
sampling file found, skipping sampling...
###################################
### TRINITY to assemble repeats ###
###################################

***** TRINITY iteration 1 *****

Selecting reads for Trinity iteration number 1...
awk: fatal : impossible d'ouvrir le fichier « /home/loutre/dnaPipeTE/try/Trinity_run0/chrysalis/readsToComponents.out.sort » en lecture (Aucun fichier ou dossier de ce type)
Done

Current settings:
time(seconds)        unlimited
file(blocks)         unlimited
data(kbytes)         unlimited
stack(kbytes)        8192
coredump(blocks)     0
memory(kbytes)       unlimited
locked memory(kbytes) 64
process              515438
nofiles              1024
vmemory(kbytes)      unlimited
locks                unlimited


Error, Trinity requires access to Java version 1.6 or 1.7.  Currently installed version is: java version "1.8.0_121"
Java(TM) SE Runtime Environment (build 1.8.0_121-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)
Trinity iteration 1 Done'
renaming Trinity output...
awk: fatal : impossible d'ouvrir le fichier « /home/loutre/dnaPipeTE/try/Trinity_run1/Trinity.fasta » en lecture (Aucun fichier ou dossier de ce type)
done
/home/loutre/dnaPipeTE/try/Annotation/one_RM_hit_per_Trinity_contigs
/home/loutre/dnaPipeTE/try/Annotation/Best_RM_annot_80-80
/home/loutre/dnaPipeTE/try/Annotation/Best_RM_annot_partial
#######################################
### REPEATMASKER to anotate contigs ###
#######################################

RepeatMasker version open-4.0.6
The RepeatMasker installation directory ($RepeatMaskerConfig::REPEATMASKER_DIR) is incorrectly set in the RepeatMaskerConfig.pm file.  Please open the RepeatMaskerConfig.pm file  and edit the $RepeatMaskerConfig::REPEATMASKER_DIR line.
Traceback (most recent call last):
  File "./dnaPipeTE.py", line 700, in <module>
    RepeatMasker(config['DEFAULT']['RepeatMasker'], args.RepeatMasker_library, args.RM_species, args.cpu, args.output_folder, args.RM_threshold)
  File "./dnaPipeTE.py", line 359, in __init__
    self.repeatmasker_run()
  File "./dnaPipeTE.py", line 378, in repeatmasker_run
    with open(self.output_folder+"/Trinity.fasta.out", 'r') as trinity_handle:
FileNotFoundError: [Errno 2] No such file or directory: '/home/loutre/dnaPipeTE/try/Trinity.fasta.out'
Loutre:~/dnaPipeTE$ pwd 
/home/loutre/dnaPipeTE

```
 I think this is a problem with java version. I already Trinity on my computer, and it worked fine with java 1.8. My trinity version is 1.3, yours is installed trinityrnaseq_r2013_08_14.

Do you think it will be possible to update trinity version ? Or should I install a older java version ?

**Comment by clemgoub** [comment]
Hi!

Sorry for my late reply... I did not get notified of your question on github...
We included this version of Trinity because the most recent one have reamed outputs that wont fit anymore the original script. Thus, I would recommend to use an older Java version for the use of dnaPipeTE.

Sorry for that. We will take that issue into consideration for the future updates.

Thanks for your feedback.

Clément
